transform/surveys/019.0018.json

```diff
@@ -1,251 +1,276 @@
+                    "question_id":"709",
+                    "number":"709",
+                    "type":"currency"
+                },
+                {
+                    "title":"What was the value of acquisitions and proceeds from disposal of capital assets for telecommunication equipment?",
+                    "text":"Value of proceeds from disposals for telecommunication equipment",
+                    "question_id":"710",
+                    "number":"710",
+                    "type":"currency"
+                }
+            ]
         },
         {
-          "title": "Please indicate the reasons for any changes in the total investment",
-          "text": "Other",
-          "question_id": "146k",
-          "number": "146k",
-          "type": "checkbox",
-          "options": [
-            "Other"
-          ]
+            "title":"Other machinery or equipment not mentioned",
+            "questions":[
+                {
+                    "title":"What was the value of acquisitions and proceeds from disposal of capital assets for other machinery or equipment not mentioned?",
+                    "text":"Value of acquisitions for other machinery or equipment",
+                    "question_id":"711",
+                    "number":"711",
+                    "type":"currency"
+                },
+                {
+                    "title":"What was the value of acquisitions and proceeds from disposal of capital assets for other machinery or equipment not mentioned?",
+                    "text":"Value of proceeds from disposals for other machinery or equipment",
+                    "question_id":"712",
+                    "number":"712",
+                    "type":"currency"
+                }
+            ]
         },
         {
-          "title": "Please describe the changes in total investment",
-          "text": "Significant changes comments",
-          "question_id": "146",
-          "number": "146",
-          "type": "contains"
+            "title":"Significant changes",
+            "questions":[
+                {
+                    "title":"Did any significant changes occur to the total investment?",
+                    "text":"Did any significant changes occur to the total investment?",
+                    "question_id":"146a",
+                    "number":"146a",
+                    "type":"radio",
+                    "options":[
+                        "Yes",
+                        "No"
+                    ]
+                },
+                {
+                    "title":"Please indicate the reasons for any changes in the total investment",
+                    "text":"Start or end of a long term project",
+                    "question_id":"146b",
+                    "number":"146b",
+                    "type":"checkbox",
+                    "options":[
+                        "Start or end of a long term project"
```

Comment:
I know that this is just the way that it's done, but it would be nicer if we could do something more sensible like...
```
"title":"Please indicate the reasons for any changes in the total investment",
                    "text":"reasons for any changes in the total investment",
                    "question_id":"146b",
                    "number":"146b",
                    "type":"checkbox",
                    "options":[
                        "Start or end of a long term project",
                        "Site changes, for example, openings, closures, refurbishments or upgrades",
                        "etc",
]}

Reply:
It's just the way its done when each option has a `q_code`. Ones that don't hav e `q_code`, ie Yes/No question, are done like you said. Maybe something to consider in the future.